pkgs/development/libraries/libtomcrypt/default.nix

```diff
@@ -21,20 +21,29 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  nativeBuildInputs = [ libtool libtommath ];
+  nativeBuildInputs = [ libtool ];
+  buildInputs = [ libtommath ];
+
+  outputs = [ "out" "dev" ];
 
   postPatch = ''
     substituteInPlace makefile.shared --replace "LIBTOOL:=glibtool" "LIBTOOL:=libtool"
   '';
 
+  env.CONFIG_CFLAGS = "-DUSE_LTM -DLTM_DESC -DLTC_PTHREAD";
+
   preBuild = ''
     makeFlagsArray+=(PREFIX=$out \
-      CFLAGS="-DUSE_LTM -DLTM_DESC -DLTC_PTHREAD" \
+      CFLAGS="$CONFIG_CFLAGS" \
       EXTRALIBS=\"-ltommath\" \
       INSTALL_GROUP=$(id -g) \
       INSTALL_USER=$(id -u))
   '';
 
+  preFixup = ''
+    sed -i "/^Cflags:/s|\$| $CONFIG_CFLAGS|" $out/lib/pkgconfig/libtomcrypt.pc
```

Comment:
In the original version this is `Cflags: -I${includedir} @MPI_PROVIDERS_CFLAGS@`, but you maybe don't need the `-I${includedir}`!?

Reply:
As long as Nix is setting everything up for cc-wrapper, it's not technically required. I think it makes sense to ship a complete, self-contained pkg-config file anyway.

Reply:
Oh, I misunderstood the sed expression :) I thought you're removing that part, but now it's clear what happens.

Reply:
`sed` expressions will do that :laughing: